hack/verify-flags/exceptions.txt

```diff
@@ -5,11 +5,13 @@ cluster/aws/templates/configure-vm-aws.sh:  # We set the hostname_override to th
 cluster/aws/templates/configure-vm-aws.sh:  api_servers: '${API_SERVERS}'
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "hostname_override"
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "runtime_config"
+cluster/aws/templates/configure-vm-aws.sh:  kubelet_api_servers: '${KUBELET_APISERVER}'
```

Comment:
strange coincidence that cluster scripts you didn't modify happen to make use of similarly named variables, and that those authors didn't run the validation scripts for their commits. i'm not super-excited about picking up those changes as part of this PR unless we fail PR validation without them.